conf/st2.conf.sample

```diff
@@ -73,8 +73,12 @@ backend = flat_file
 home_dir = /opt/cslang
 
 [content]
+# A URL pointing to the pack index. StackStorm Exchange is used by default. Use a comma-separated list for multiple indexes if you want to get other packs discovered with "st2 pack search".
+index_url = https://index.stackstorm.org/v1/index.json # comma separated list allowed here.
```

Comment:
How does the ordering work if the same pack is available on multiple indexes? Does it work from left to right aka first one has priority?

It would be good to document this behavior somewhere in the docs.